_progress-bar.scss

```diff
@@ -0,0 +1,279 @@
+    &:not(:empty) {
+        position: relative;
+
+        width: 45px;
+        height: 45px;
+        margin-right: 10px;
+    }
+}
+
+// Radial Progress Bar Background
+// ---
+
+.c-progress-bar__radial-background {
+    stroke: $progress-bar__background-color;
+}
+
+// Radial Progress Bar Progress
+// ---
+//
+// Used to show the current level of progress
+
+.c-progress-bar__radial-progress {
+    .c-progress-bar.c--progress & {
+        stroke: $progress-bar__progress-color;
+    }
+
+    .c-progress-bar.c--success & {
+        stroke: $progress-bar__success-color;
+    }
+
+    .c-progress-bar.c--error & {
+        stroke: $progress-bar__error-color;
+    }
+}
+
+// Radial Progress Bar SVG
+// ---
+//
+// Where the magic happens
+
+.c-progress-bar__svg {
+    .c-progress-bar.c--direction-rtl & {
+        transform: scaleX(-1);
+    }
+}
+
+
+
+// Progress Bar Middle
+// ---
+//
+// Holds the linear progress bar and the label
+// Needed as a separate element so we can flexbox everything nicely
+
+.c-progress-bar__middle {
+    flex-grow: 1;
+    margin-right: 5px;
+}
+
+
+
+// Linear Progress Bar Background
+// ---
+
+.c-progress-bar__linear-background {
+    position: relative;
+
+    overflow: hidden;
+    height: 5px;
+
+    background-color: $progress-bar__background-color;
+
+    .c-progress-bar.c--direction-rtl & {
+        transform: scaleX(-1);
+    }
+}
+
+// Linear Progress Bar Progress
+// ---
+//
+// Used to show the current level of progress
+
+.c-progress-bar__linear-progress {
+    position: absolute;
+    transform: translate3d(-100%, 0, 0);
+
+    height: 5px;
+    width: 100%;
+
+    transition: transform 0.5s ease-in;
+
+    .c-progress-bar.c--progress & {
+        background-color: $progress-bar__progress-color;
+    }
+
+    .c-progress-bar.c--success & {
+        background-color: $progress-bar__success-color;
+    }
+
+    .c-progress-bar.c--error & {
+        background-color: $progress-bar__error-color;
+    }
+}
+
+
+
+// Progress Bar Label
+// ---
+//
+// Displays a user defined label which describes the state of progress
+
+.c-progress-bar__label {
+    .c-progress-bar.c--progress & {
+        color: $progress-bar__progress-color;
+    }
+
+    .c-progress-bar.c--success & {
+        color: $progress-bar__success-color;
+    }
+
+    .c-progress-bar.c--error & {
+        color: $progress-bar__error-color;
+    }
+}
+
+
+
+// Extra section
+// ---
+//
+// This section is displayed beside the horizontal bar and is used to show the progress indicator text or icon
+// 1. When the radial progress bar is shown, we want to hide this section
+
+.c-progress-bar__extra {
+    position: relative;
+
+    width: 35px;
+    height: 25px;
+
+    .c-progress-bar.c--radial & {
+        display: none;
+    }
+}
+
+
+
+// Indicator
+// ---
+//
+// This section contains the current value of the progress bar and the indicator icons
+
+.c-progress-bar__indicator {
+    width: 25px;
+    height: 25px;
+
+    text-align: center;
```

Comment:
Disabling the above `width`, `height` and `text-align` seems to not do anything. What are these intended to do?